### DISCUSSION TITLE
cuRobo solves IK but not Motions for custom robot -- consider warning about invalid velocity limits?

### DISCUSSION THREAD

**Discussion by peter-mitrano-bg** [NVlabs/curobo#229] [Software Q&A]
Hello! I was debugging why MotionGen was failing to solve for my robot, even though IK was working. After a while I figured out it was because my URDF doesn't have valid velocity (or effort, but I don't think that's used) limits, they were just 0. This caused the LFBGS solver to explode and the cost become NaN.

I would suggest putting a warning in the code to mitigate this issue, since not all URDFs are perfect. For example in `bound_cost.py`, perhaps at configuration time to avoid slowing down the solver?

I can try to make a pull request, if someone has the time to review & merge it :slightly_smiling_face:

**Comment by balakumar-s** [comment]
It's been added here: https://github.com/NVlabs/curobo/blob/7196be75f5294f0997a2b55ed9f24a088d57b33e/src/curobo/rollout/cost/bound_cost.py#L59
